perllib/FixMyStreet/App/Controller/Contact.pm

```diff
@@ -106,7 +110,13 @@ sub determine_contact_type : Private {
             $c->stash->{rejecting_report} = 1;
         }
     } elsif ( $c->cobrand->abuse_reports_only ) {
-        $c->detach( '/page_error_404_not_found' );
+        # General enquiries replaces contact form if enabled
+        if ( $c->cobrand->can('setup_general_enquiries_stash') ) {
+            $c->res->redirect( '/contact/enquiry' );
+            return 1;
```

Comment:
This should detach, otherwise processing continues as normal (I realise the `auto`s and `/contact/index` don't do much, but nevertheless)